frontend/app/routes/public/renew/$id/adult-child/children/$childId/information.tsx

```diff
@@ -202,6 +224,7 @@ export default function RenewFlowChildInformation() {
     dateOfBirthYear: 'date-picker-date-of-birth-year',
     clientNumber: 'client-number',
     isParent: 'input-radio-is-parent-radios-option-0',
+    notFound: 'not-found',
```

Comment:
FYI there is currently no `id` in this component whose value is `not-found`, so the error message link will be broken.

Reply:
Removed the ID